src/pages/CommitDetailPage/queries/CommitPageDataQueryOpts.tsx

```diff
@@ -169,6 +173,7 @@ export const useCommitPageData = ({
                 </p>
               ),
             },
+            dev: 'CommitPageDataQueryOpts - 403 Owner not activated',
```

Comment:
Do we need to populate `error` to have that sent to Sentry? May be nice to have a copy of the `parsedData` perhaps